cmd/graphql2go/main.go

```diff
@@ -870,8 +872,9 @@ func (g *generator) genObjectDefinition(def *ast.ObjectDefinition) {
 		g.printf("\t},\n")
 	}
 	g.printf("\tFields: graphql.Fields{\n")
-	for fieldName, fieldDefName := range fieldDefNamesByFieldName {
-		g.printf("\t%q: %s,\n", fieldName, fieldDefName)
+	sort.Strings(fieldNames)
+	for _, fieldName := range fieldNames {
```

Comment:
Can't we just iterate `for _, f := range def.Fields` here? since we're using that to populate the map anyway `fieldDefNamesByFieldName[f.Name.Value]`

Reply:
Good callout, that works better too with low overhead

Reply:
Done